package/kedro_viz/api/rest/responses.py

```diff
@@ -19,6 +28,11 @@ class Config:
         orm_mode = True
 
 
+class UserCredentials(BaseModel):
```

Comment:
can we have this more specific like AWSCredentials (since we might have different deployers)